jerry-core/vm/vm.c

```diff
@@ -136,8 +144,20 @@ vm_op_set_value (ecma_value_t object, /**< base object */
 
     if (ECMA_IS_VALUE_ERROR (to_object))
     {
+      ecma_free_value (to_object);
+
+#ifdef JERRY_ENABLE_ERROR_MESSAGES
+      ecma_string_t *error_msg_p = ecma_format_ecma_values ("Cannot set property '%' of %", property, object);
+      ecma_object_t *error_obj_p = ecma_new_standard_error_with_message (ECMA_ERROR_TYPE, error_msg_p);
+      ecma_value_t error_value = ecma_make_error_obj_value (error_obj_p);
+      ecma_deref_ecma_string (error_msg_p);
```

Comment:
However, if we add a helper function in "ecma-exceptions.c" to raise ecma-error, why not merge `ecma_format_ecma_values` into that function?